proto/agglayer/node/types/v1/network_state.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package agglayer.node.types.v1;
+
+import "agglayer/interop/types/v1/bytes.proto";
+import "agglayer/node/types/v1/certificate_id.proto";
+import "agglayer/node/types/v1/certificate_header.proto";
+
+// NetworkState represents the current state of a network in the AggLayer.
+message NetworkState {
+  // The current status of the network (e.g., "active", "syncing", "error").
+  string network_status = 1;
```

Comment:
This could be an enum as defined [here](https://github.com/agglayer/agglayer/pull/977#discussion_r2307910995)